spring-data-multi-storage-transaction-sample/README.md

```diff
@@ -8,77 +8,68 @@ This tutorial describes how to create a sample Spring Boot application by using
+Since they are available under a commercial license, you need to get the license and permission to access them.
+For more details, please [contact us](https://scalar-labs.com/contact_us/).
+
+You also need the `gpr.user` property for your GitHub username and the `gpr.key` property for your personal access token.
+So you need to have the properties in `~/.gradle/gradle.properties`, or specify the properties with the `-P` option when running the `./gradlew` command as follows:
+
+```shell
+$ ./gradlew run ... -Pgpr.user=<your GitHub username> -Pgpr.key=<your personal access token>
+````
+
+Please read [Install - ScalarDB SQL](https://github.com/scalar-labs/scalardb-sql#install) for more detail.
+
 ## Sample application
 
 ### Overview
 
 This tutorial describes how to create a sample Spring Boot application for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) but by using Spring Data JDBC for ScalarDB with Multi-storage Transaction.
 Please note that application-specific error handling, authentication processing, etc. are omitted in the sample application since this tutorial focuses on explaining how to use Spring Data JDBC for ScalarDB with Multi-storage Transaction.
-For details, please see [Guide of Spring Data JDBC for ScalarDB](https://scalar-labs.github.io/scalardb-sql/spring-data-guide.html).
+For details, please see [Guide of Spring Data JDBC for ScalarDB](https://github.com/scalar-labs/scalardb-sql/blob/main/docs/spring-data-guide.md).
 
 ![Overview](images/overview.png)
 
 ### Schema
 
-[The schema](schema.json) is as follows:
-
-```json
-{
-  "customer.customers": {
-    "transaction": true,
-    "partition-key": [
-      "customer_id"
-    ],
-    "columns": {
-      "customer_id": "INT",
-      "name": "TEXT",
-      "credit_limit": "INT",
-      "credit_total": "INT"
-    }
-  },
-  "order.orders": {
-    "transaction": true,
-    "partition-key": [
-      "customer_id"
-    ],
-    "clustering-key": [
-      "timestamp"
-    ],
-    "secondary-index": [
-      "order_id"
-    ],
-    "columns": {
-      "order_id": "TEXT",
-      "customer_id": "INT",
-      "timestamp": "BIGINT"
-    }
-  },
-  "order.statements": {
-    "transaction": true,
-    "partition-key": [
-      "order_id"
-    ],
-    "clustering-key": [
-      "item_id"
-    ],
-    "columns": {
-      "order_id": "TEXT",
-      "item_id": "INT",
-      "count": "INT"
-    }
-  },
-  "order.items": {
-    "transaction": true,
-    "partition-key": [
-      "item_id"
-    ],
-    "columns": {
-      "item_id": "INT",
-      "name": "TEXT",
-      "price": "INT"
-    }
-  }
-}
+[The schema](schema.sql) is as follows:
+
+```sql
+CREATE COORDINATOR TABLES IF NOT EXIST;
+
+CREATE NAMESPACE IF NOT EXISTS sample;
+
+CREATE TABLE IF NOT EXISTS customer.customers (
+customer_id INT PRIMARY KEY,
```

Comment:
Some indent would be better, I think.

Reply:
Good catch! Fixed in dd7ea57cb3e09b7ed2986915b511087b6988bb40. Thanks.